book/en/src/by-example/starting_a_project.md

```diff
@@ -1,6 +1,9 @@
 # Starting a new project
 
-A recommendation when starting a RTIC project from scratch is to follow RTIC's [`defmt-app-template`].
+A recommendation when starting a RTIC project from scratch on an ARMv7-M or ARMv8-M-main MCU is to 
```

Comment:
`defmt` isn't as trivial to setup in ARMv6-M since the `#[global_logger]` can't use RTT or ITM, unless I missed something when trying to add it to my project.

Reply:
Interesting, I did not know there were issues with RTT or ITM on thumbv6m. Granted, I have not tried building/running that yet, but I've seen that the [rp2040 seems to support defmt](https://github.com/rp-rs/rp2040-project-template)

My rp2040 devices are still in transit, hopefully here before xmas :smile: 

@korken89 I believe you might have some insight into this?

Reply:
`defmt` works fine on `thumbv6`.

Reply:
I'm not super familiar with the debug interfaces, but from what I could tell, RTT requires JTAG (not SWD as described by Cortex-M0+ datasheets). Alternatively, ITM requires a core module not present in the ARMv6m architecture. Which `#[global_logger]` configuration works? Mostly like to know so I can add it to my project lol.

Reply:
Huh, well I just tried defmt-rtt and it compiled under thumbv6, so that clears this up.